README.md

```diff
@@ -72,3 +72,4 @@ Following is the latest doxygenize version:
 | Release version | Docker image | Tag aliases |
 | --------------- | ------------ | ----------  |
 | v1.7.0 | Alpine 3.15.4 | v1, v1.7 |
+| v1.7.0 | Alpine 3.22.0 | v1, v1.7 |
```

Comment:
Thanks for the update. Please bump the version to v1.7.1. I will merge it then. Test the action and if it passes, I will officially release the new version.

Reply:
Is it now as you want it?

Reply:
Yes!